backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanSerializerExtension.kt

```diff
@@ -24,11 +25,12 @@ internal class KonanSerializerExtension(val context: Context, override val metad
     val inlineDescriptorTable = DescriptorTable(context.irBuiltIns)
     override val stringTable = KonanStringTable()
     override fun shouldUseTypeTable(): Boolean = true
+    internal val sourceFileMap = SourceFileMap()
```

Comment:
Is this correct? `KonanSerializerExtension` seems to be created per package fragment, and declarations of one source file can be split into multiple package fragments.

Reply:
Same as above.